### PR TITLE
[issue #2366]Remove warning code in [shenyu-examples]

### PR DESCRIPTION
**Pull request by li-keguo**
about issue #2366 

work list :

- [x] clean warning code in shenyu-examples-websocket
- [x] clean warning code in shenyu-examples-tars [not found]
- [x] clean warning code in shenyu-examples-springcloud
- [x] clean warning code in shenyu-examples-sofa
- [x] clean warning code in shenyu-examples-plugin
- [x] clean warning code in shenyu-examples-motan [not found]
- [x] clean warning code in shenyu-examples-grpc
- [x] clean warning code in shenyu-examples-eureka [not found]

 shenyu-examples-dubbo give it to @dizent , if not found, I will fix it

### CODE REVIEW

File: shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java [hunk 1]
```diff
@@ -69,7 +69,8 @@ public void onNext(final RequestData value) {
 
             @Override
             public void onError(final Throwable t) {
-
+                System.err.println(t.getMessage());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java:72]: I think log file is ok, and no need to print to system.err

File: shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java [hunk 2]
```diff
@@ -69,7 +69,8 @@ public void onNext(final RequestData value) {
...
+                t.printStackTrace();
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java:73]: how about use log

File: shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java [hunk 3]
```diff
@@ -99,7 +100,8 @@ public void onNext(RequestData value) {
             }
 
             @Override
-            public void onError(final Throwable t) { t.printStackTrace();
+            public void onError(final Throwable t) {
+                t.printStackTrace();
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-grpc/src/main/java/org/apache/shenyu/examples/grpc/stream/StreamServiceImpl.java:104]: how about use log

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 4]
```diff
@@ -33,57 +33,57 @@
     public void handlerPlugin(final PluginData pluginData) {
         System.out.println("hello, im extend plugin dataHandler" );
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:34]: how about use log to replace system.err

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 5]
```diff
@@ -33,57 +33,57 @@
...
+
     /**
      * Remove rule.
      *
      * @param ruleData the rule data
      */
     @Override
     public void removeRule(final RuleData ruleData) {
-        
+        System.out.println("rule data removed: name = " + ruleData.getName());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:84]: how about use log to replace system.out

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 6]
```diff
@@ -33,57 +33,57 @@
...
+
     /**
      * Handler rule.
      *
      * @param ruleData the rule data
      */
     @Override
     public void handlerRule(final RuleData ruleData) {
-        
+        System.out.println("rule processing : name = " + ruleData.getName());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:74]: how about use log to replace system.out

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 7]
```diff
@@ -33,57 +33,57 @@
...
+
     /**
      * Remove selector.
      *
      * @param selectorData the selector data
      */
     @Override
     public void removeSelector(final SelectorData selectorData) {
-        
+        System.out.println("selector removed : name = " + selectorData.getName());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:64]: how about use log to replace system.out

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 8]
```diff
@@ -33,57 +33,57 @@
...
+
     /**
      * Handler selector.
      *
      * @param selectorData the selector data
      */
     @Override
     public void handlerSelector(final SelectorData selectorData) {
-        
+        System.out.println("selector processing : name = " + selectorData.getName());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:54]: how about use log to replace system.out

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java [hunk 9]
```diff
@@ -33,57 +33,57 @@
...
+
     /**
      * Remove plugin.
      *
      * @param pluginData the plugin data
      */
     @Override
     public void removePlugin(final PluginData pluginData) {
-        
+        System.out.println("selector removed : name = " + pluginData.getName());
```
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:44]: how about use log to replace system.out

File: shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java (hunk 4 above)
Comment (AhahaGe) [inline shenyu-examples/shenyu-examples-plugin/src/main/java/org/apache/shenyu/examples/plugin/ext/ExtPluginDataHandler.java:34]: how about use log to replace system.out

**Comment by li-keguo** [comment]
ok e......  Just say it once

**Review by yu199195 — APPROVED** [review]

**Pull request merged**